tests/python_client/testcases/test_partition.py

```diff
@@ -498,16 +499,15 @@ def test_partition_release(self):
         assert len(res1) == 1
 
         # release the first partition
-        partition_w1.release()
```

Comment:
I think this case should not be modified directly, release all partitions is a scenario too. Suggest to modify the comments and the case name like test_partition_release_all for this case, and add a new case to cover the scenario  of release one partition only.